awscli/data/ec2/2016-11-15/completions-1.json

```diff
@@ -1701,6 +1701,781 @@
+          }
+        ]
+      }
+    },
+    "DescribeLaunchTemplates": {
+      "LaunchTemplateIds": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "LaunchTemplate",
+            "resourceIdentifier": "LaunchTemplateId"
+          }
+        ]
+      },
+      "LaunchTemplateNames": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "LaunchTemplateVersion",
+            "resourceIdentifier": "LaunchTemplateName"
+          }
+        ]
+      }
+    },
+    "DescribeNatGateways": {
+      "NatGatewayIds": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "NatGateway",
+            "resourceIdentifier": "NatGatewayId"
+          }
+        ]
+      }
+    },
+    "DescribeNetworkAcls": {
+      "NetworkAclIds": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "NetworkAcl",
+            "resourceIdentifier": "NetworkAclId"
+          }
+        ]
+      }
+    },
+    "DescribeNetworkInterfaceAttribute": {
+      "NetworkInterfaceId": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "NetworkInterface",
+            "resourceIdentifier": "NetworkInterfaceId"
+          }
+        ]
+      }
+    },
+    "DescribeNetworkInterfacePermissions": {
+      "NetworkInterfacePermissionIds": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "NetworkInterfacePermission",
+            "resourceIdentifier": "NetworkInterfacePermissionId"
+          }
+        ]
+      }
+    },
+    "DescribeNetworkInterfaces": {
+      "NetworkInterfaceIds": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "NetworkInterface",
+            "resourceIdentifier": "NetworkInterfaceId"
+          }
+        ]
+      }
+    },
+    "DescribePlacementGroups": {
+      "GroupNames": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "PlacementGroup",
+            "resourceIdentifier": "GroupName"
+          }
+        ]
+      }
+    },
+    "DescribePrefixLists": {
+      "PrefixListIds": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "PrefixList",
+            "resourceIdentifier": "PrefixListId"
+          }
+        ]
+      }
+    },
+    "DescribePrincipalIdFormat": {
+      "Resources": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "IdFormat",
+            "resourceIdentifier": "Resource"
+          }
+        ]
+      }
+    },
+    "DescribeRegions": {
+      "RegionNames": {
+        "completions": [
+          {
+            "parameters": {},
+            "resourceName": "Region",
+            "resourceIdentifier": "RegionName"
```

Comment:
Hmmm for this one, I was not able to get the completions. Any idea on why? Are you able to get completions for `--region-names`?

Reply:
Yeah it works for me, you'll need to regenerate the cache file.  Whenever new completions-1.json files are added/updated, you have to rerun `./scripts/gen-ac-index` again.